camera_calibration_parsers/CMakeLists.txt

```diff
@@ -4,11 +4,8 @@ project(camera_calibration_parsers)
 find_package(catkin REQUIRED sensor_msgs rosconsole roscpp roscpp_serialization)
 
 find_package(PythonLibs REQUIRED)
-if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
-  find_package(Boost REQUIRED COMPONENTS filesystem python)
-else()
-  find_package(Boost REQUIRED COMPONENTS filesystem python3)
-endif()
+find_package(PythonInterp REQUIRED)
```

Comment:
This should probably use `find_package(PythonInterp ${PYTHON_VERSION} REQUIRED)` like Catkin does in https://github.com/ros/catkin/blob/86439ec5d2010d5c47c30b815aa97e525037930f/cmake/python.cmake#L4

Reply:
And
```cmake
find_package(PythonInterp REQUIRED)
```
should be called before
```cmake
find_package(PythonLibs REQUIRED)
```
according to a comment in the latter:
```cmake
# If calling both ``find_package(PythonInterp)`` and
# ``find_package(PythonLibs)``, call ``find_package(PythonInterp)`` first to
# get the currently active Python version by default with a consistent version
# of PYTHON_LIBRARIES.
```
and [here](https://cmake.org/cmake/help/git-master/module/FindPythonLibs.html#findpythonlibs).

Reply:
Thank you for the review, but I have the feeling there is no more interest in supporting this package from ROS. Feel free to suggest a change by editing the file, but I don't feel it is going to be merged anyway :)

Reply:
While it does look kind of abandoned, I got this package from building ROS from source alongside a couple of other packages, so I think it's still being used.